Hannah-Sten/TeXiFy-IDEA
# Introduce postfix templates

#### Summary of additions and changes

Add support for postfix templates. Ideas for more templates are welcome.

#### How to test this pull request

* Go to *Settings > Editor > General > Postfix Completion* and select the LaTeX list to see the available postfix templates.
* Try the postfix templatex in a tex file.

#### Wiki


- [x] Updated the wiki: https://github.com/Hannah-Sten/TeXiFy-IDEA/wiki/Postfix-templates
